packages/react-native/React/Views/UIResponder+React.m

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "UIResponder+React.h"
+
+static __weak id reactCurrentFirstResponder;
+
+@implementation UIResponder (React)
++ (id)reactCurrentFirstResponder
+{
+  reactCurrentFirstResponder = nil;
+  [[UIApplication sharedApplication] sendAction:@selector(reactFindFirstResponder:) to:nil from:nil forEvent:nil];
+  return reactCurrentFirstResponder;
+}
+
+- (void)reactFindFirstResponder:(id)sender
+{
+  reactCurrentFirstResponder = self;
+}
```

Comment:
This is something we used to have and we removed it. It is seen as an anti-pattern, so we'd rather not have it. Probably @javache has more context on why it got removed.

Reply:
@cipolleschi I changed another way to find first responder :)

Reply:
I found why: the previous API is a private API https://github.com/facebook/react-native/pull/37766#discussion_r1222735097